src/gui/editors/PianoRoll.cpp

```diff
@@ -64,6 +64,11 @@
 #include "TimeLineWidget.h"
 #include "StepRecorderWidget.h"
 
+//for random
```

Comment:
We already perform srand() in main.cpp and that should be enough for all of LMMS.